src/games/calculator.js

```diff
@@ -0,0 +1,20 @@
+import { getAnswer, askQuestion } from '../cli.js';
+import { operators, generateRandomNumber } from '../utils/math.js';
+
+const calculator = () => {
+  const operator = Object.keys(operators)[generateRandomNumber(Object.keys(operators).length - 1)];
```

Comment:
Too much stuff on a single line.
```suggestion
  const operatorKeys = Object.keys(operators)
  const lastOperatorIndex = Object.keys(operators).length - 1)
  const randomOperatorIndex = generateRandomNumber(lastOperatorIndex)
  const operator = operatorKeys[randomOperatorIndex]
```

Reply:
but if you will introduce `getRandomElement` function it will become
```suggestion
  const operator = getRandomElement(Object.keys(operators))
```